fala/templates/adviser/accessibility_statement.html

```diff
@@ -0,0 +1,117 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="govuk-grid-column-two-thirds">
+    <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+
+    <h1 class="govuk-heading-xl">Accessibility statement for Find a legal advisor</h1>
+
+    <p class="govuk-body">
+      This accessibility statement applies to the ‘Find a legal advisor’ service.
+    </p>
+
+    <p class="govuk-body">
+      This website is run by the <a class="govuk-link" href="https://www.gov.uk/government/organisations/legal-aid-agency" target="_blank" rel="noopener external noreferrer">Legal Aid Agency</a> (<abbr>LAA</abbr>), part of the Ministry of Justice (<abbr>MoJ</abbr>).
```

Comment:
Checked with content designer: _the links within the accessibility statement and privacy policy should not open in a new tab._